components/net/http_cache.rs

```diff
@@ -0,0 +1,486 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#![deny(missing_docs)]
+
+//! A memory cache Implements the logic specified in http://tools.ietf.org/html/rfc7234
```

Comment:
`A memory cache that implements the logic specified in <http://tools.ietf.org/html/rfc7234>`